diff --git a/include/configs/armada_38x.h b/include/configs/armada_38x.h
index 75b8bf5..696e496 100644
--- a/include/configs/armada_38x.h
+++ b/include/configs/armada_38x.h
@@ -718,4 +718,7 @@ extern int nand_get_env_offs(void);
 
 #define CONFIG_STACKSIZE        (1 << 20)       /* regular stack - up to 4M (in case of exception)*/
 
+// support non-mkimage'd ramdisks
+#define CONFIG_SUPPORT_RAW_INITRD
+
 #endif /* __CONFIG_H */
